main.go

```diff
@@ -70,7 +69,7 @@ func main() {
 	log.SetFlags(log.Ldate | log.Ltime | log.Lmicroseconds)
 	if strings.ToLower(os.Getenv("ENABLE_GCSFUSE_VIPER_CONFIG")) == "true" {
 		// TODO: implement the mount logic instead of simply returning nil.
-		rootCmd, err := cmd.NewRootCmd(func(*cfg.Config, string, string) error { return nil })
+		rootCmd, err := cmd.NewRootCmd(cmd.Mount)
```

Comment:
Should we move profiling handlers to this main now: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/cmd/legacy_main.go#L481-L483

Reply:
Good catch!